src/views/Activate/ActivateCode.js

```diff
@@ -36,12 +36,11 @@ export default function ActivateCode() {
     setDerivedPoint,
   } = useActivateFlow();
 
-  const ticketInput = useTicketInput({
+  const [ticketInput, { pass: validTicket }] = useTicketInput({
```

Comment:
it is pretty standard for hooks, since we frequently need to rename generic arguments to contextual ones (like `const [myState, setMyState] = useState()`)

in this case I'm just making it easier/more semantic to do the "create the input state and also give me a local reference to some of it" pattern that we were doing in every instance of an input. the destructuring & renaming syntax can be verbose, but I don't know that there's a more useful alternative